src/test/java/eu/merloteducation/organisationsorchestrator/ParticipantServiceTests.java

```diff
@@ -568,13 +587,49 @@ void createParticipantWithValidRegistrationForm() throws Exception {
         assertEquals(0,  varArgs.getValue().getConnectors().size());
     }
 
+    @Test
+    void createParticipantWithValidRegistrationFormAsFederator() throws Exception {
```

Comment:
Ist der Test nicht derselbe wie oben? Oben ist die Rolle FedAdmin von someorga und in dem Test FedAdmin von somefedorga